frontend/packages/topology/src/components/page/TopologyView.tsx

```diff
@@ -386,7 +392,13 @@ export const ConnectedTopologyView: React.FC<ComponentProps> = ({
                         </span>
                       </div>
                     )}
-                    {viewContent}
+                    {nodesLength && nodesLength > 99 && !showTopologyAnyway ? (
```

Comment:
Use a const for max nodes.
```suggestion
                    {nodesLength >= MAX_NODES && !showTopologyAnyway ? (
```